app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/FullDetailsFragment.java

```diff
@@ -677,8 +677,18 @@ public void setTitle(String title) {
     }
 
     private String getRunTime() {
-        Long runtime = Utils.getSafeValue(mBaseItem.getRunTimeTicks(), mBaseItem.getRunTimeTicks());
-        return runtime != null && runtime > 0 ? String.format("%d%s", (int) Math.ceil((double) runtime / 600000000), getString(R.string.lbl_min)) : "";
+        Long runtime_ticks = Utils.getSafeValue(mBaseItem.getRunTimeTicks(), mBaseItem.getRunTimeTicks());
```

Comment:
Variables need to be camelCase, so:

```suggestion
        Long runtimeTicks = Utils.getSafeValue(mBaseItem.getRunTimeTicks(), mBaseItem.getRunTimeTicks());
```

etc.

Reply:
Fixed in [88ad0e0](https://github.com/jellyfin/jellyfin-androidtv/pull/3878/commits/88ad0e07b99fa034edd2dafb160d54bff0667815).